webdriver/tests/bidi/script/add_preload_script/contexts.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+
+from webdriver.bidi.modules.script import ContextTarget
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("domain", ["", "alt"], ids=["same_origin", "cross_origin"])
+async def test_add_preload_script_in_frame_with_top_context_specified(
```

Comment:
There is no need for the command's name as prefix. That is already part of the folder name. Also we won't have a test in this file which specifies child browsing contexts so the following name should be fine (similar to other files):

```suggestion
async def test_iframe(
```

Also I would rather see that as a general test not necessarily bound to the contexts argument so being part of `add_preload_script.py`?

Reply:
I am not sure what is the requirement here is - we do have a test in `invalid.py` that check that you can't supply non-top-level context. 
https://github.com/web-platform-tests/wpt/pull/41932/files#diff-e56c62f32fa10718d53bcb4f6e85a62d4b6cc241850230871d04226006473708R225

Reply:
Yeah we probably need a better name here. What about `test_traverse_to_iframe`?

Reply:
I don't have a strong opinion here, neither `_iframe` and `_traverse_to_iframe` sound perfect, so happy to accept any changes. Please open a new suggestion for what you feel is appropriate and I will accept it though the UI.